### ISSUE TITLE
copy message not working

### ISSUE THREAD

**Issue by carepack** [aebruno/whisperfish#35]
hi @aebruno,
when I'm long pressing a message in a chat I can select copy or delete. Selecting copy has no effect. text is not in the clipboard to padte somewhere.

ahoy

**Comment by d9h02f** [comment]
Works for me, I think it's a problem with the keyboard itself which sometimes doesn't let you copy anything anymore, and only fully restarting the device solves it.

**Comment by carepack** [comment]
no. I can copy anywhere else. Which version you're running? I'm on the latest commit and it isn't working there.

**Comment by carepack** [comment]
but @aebruno I'm not on the latest textsecure commit on device-provisioning branch. will update and report.

**Comment by carepack** [comment]
ok,

tested with the latest commits on whisperfish and textsecure. copy is not working. Nothing in the clipboard.

**Comment by aebruno** [comment]
This appears to be an issue with sailfish. I've experienced this with other applications. See https://together.jolla.com/question/832/system-wide-clipboard-copy-paste-capability/

Looks like other apps have a work around and will try to implement a fix.
